test/screens/play/play_page_test.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:ui';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:treasure_of_the_high_seas/screens/play/play_page.dart';
+import 'package:treasure_of_the_high_seas/screens/play/player_hand.dart';
+import 'package:treasure_of_the_high_seas/screens/play/scrying_page.dart';
+
+import '../../test_utils.dart';
+
+void main() {
+  testWidgets('should display the ScryPage if there are cards being scryed',
+      (WidgetTester tester) async {
+    //ugly hack to set a stupid size, so this test doesn't complain about random overflows
+    tester.binding.window.physicalSizeTestValue = Size(2000, 1920);
```

Comment:
This is probably a TODO to come back to. Not sure why the test was unhappy but it suggests the layout of the scrying page needs work to work properly on different screen sizes (though it looks fine to me in the emulator...)